rspace/src/main/scala/coop/rchain/rspace/pure/Store.scala

```diff
@@ -0,0 +1,75 @@
+package coop.rchain.rspace.pure
+
+import coop.rchain.rspace.internal.{Datum, Row, WaitingContinuation}
+
+import scala.collection.immutable.Seq
+
+/** The interface for the underlying store
+	*
+	* @tparam F a type constructor representing an effect (for example ReaderT[F, LMDBContext, ?])
+	* @tparam C a type representing a channel
+	* @tparam P a type representing a pattern
+	* @tparam A a type representing an arbitrary piece of data
+	* @tparam K a type representing a continuation
+	*/
+trait Store[F[_], C, P, A, K] {
+
+  /**
+		* The type of transactions
```

Comment:
Re-indent.  Did some tabs sneak in here?

Reply:
Scalafmt is configured to kill tabs. But something is wrong with it.